src/metabase/util.clj

```diff
@@ -113,6 +113,22 @@
   java.sql.Timestamp     (->iso-8601-datetime [this timezone-id] (time/unparse (ISO8601Formatter timezone-id) (coerce/from-sql-time this)))
   org.joda.time.DateTime (->iso-8601-datetime [this timezone-id] (time/unparse (ISO8601Formatter timezone-id) this)))
 
+(def ^:private time-formatter
```

Comment:
This is not super important but for functions defined by `def` it's nice to add `:arglists` metadata so when you're playing around in the CIDER repl it will show you what args it expects in the minibuffer. e.g.

```clojure
(def ^:private ^{:arglists '([timezone-id])} time-formatted
  ...) 
```

Normally I wouldn't even bother to suggest this but since it's in `metabase.util` it might be worth it to go back and add it